### PR TITLE
Dry setup

### PR DESCRIPTION
**Pull request by maurolepore**

### CODE REVIEW

File: web_tool_script_1.R [hunk 1]
```diff
@@ -27,15 +27,7 @@ source("0_global_functions.R")
...
-if (rstudioapi::isAvailable()) {
-  portfolio_name_ref_all <- c("TestPortfolio_Input") # must be the same name as in the _PortfolioParameters.yml
-  working_location <- here::here()
-  set_web_parameters(file_path = paste0(working_location,"/parameter_files/WebParameters_2dii.yml"))
-} else {
-  portfolio_name_ref_all = get_portfolio_name()
-  working_location <- getwd()
-  set_web_parameters(file_path = paste0(working_location,"/parameter_files/WebParameters_docker.yml"))
-  }
+setup_project()
```
Comment (maurolepore) [inline web_tool_script_1.R:30]: This is my only change. The rest seems to be related to white space.

File: web_tool_script_2.R [hunk 2]
```diff
@@ -20,15 +20,7 @@ source("0_portfolio_test.R")
...
-if (rstudioapi::isAvailable()) {
-  portfolio_name_ref_all <- c("TestPortfolio_Input") # must be the same name as in the _PortfolioParameters.yml
-  working_location <- here::here() 
-  set_web_parameters(file_path = paste0(working_location,"/parameter_files/WebParameters_2dii.yml"))
-} else {
-  portfolio_name_ref_all = get_portfolio_name()
-  working_location <- getwd()
-  set_web_parameters(file_path = paste0(working_location,"/parameter_files/WebParameters_docker.yml"))
-}
+setup_project()
```
Comment (maurolepore) [inline web_tool_script_2.R:23]: This is my only change; the rest is automatic changes regarding white space.

File: web_tool_script_3.R [hunk 3]
```diff
@@ -1,32 +1,24 @@
...
-if (rstudioapi::isAvailable()) {
-  portfolio_name_ref_all <- c("TestPortfolio_Input") # must be the same name as in the _PortfolioParameters.yml
-  working_location <- here::here() 
-  set_web_parameters(file_path = paste0(working_location,"/parameter_files/WebParameters_2dii.yml"))
-} else {
-  portfolio_name_ref_all = get_portfolio_name()
-  working_location <- getwd()
-  set_web_parameters(file_path = paste0(working_location,"/parameter_files/WebParameters_docker.yml"))
-}
+setup_project()
```
Comment (maurolepore) [inline web_tool_script_3.R:21]: This is my only change; the rest is automatic changes regarding white space

**Review by maurolepore — COMMENTED** [review]
Here are some comments to direct your attention to the most interesting chages.

File: dry/set_portfolio-name-ref-all_working-location_and_web-parameters.R [hunk 4]
```diff
@@ -0,0 +1,10 @@
+in_transitionmonitor <- function() grepl("^/bond", here::here())
```
[the remark this answers is not in the document]
Comment (jacobvjk) [reply dry/set_portfolio-name-ref-all_working-location_and_web-parameters.R]: it should be `/bound` though

**Comment by maurolepore** [comment]
I now tested this on docker and it seems to work as expected (although still not producing the correct index.html -- but that problem predates this PR).

**Comment by maurolepore** [comment]
I'll go ahead and merge. Jacob already gave feedback, and the integration test is passing.

**Comment by jacobvjk** [comment]
LGTM

**Pull request merged**